README.md

```diff
@@ -26,10 +32,16 @@ These instructions will get you a copy of the project up and running on your loc
 1. Run the installation script. `npm install` (If you see `babel-node: command not found`, you've missed this step.)
 1. Generate Canvas API token and copy it to clipboard.
```

Comment:
In case the user is not familiar with generating a Canvas API token, we could add a hyperlink such as, "[Generate Canvas API token](https://learninganalytics.ubc.ca/guides/get-started-with-the-canvas-api/) and copy it to clipboard."